src/test/kotlin/com/google/firebase/analytics/FirebaseAnalytics.kt

```diff
@@ -22,7 +25,15 @@ class FirebaseAnalytics {
     fun setCurrentScreen(currentActivity: Activity?, screenName: String?, classOverride: String?) {
         currentScreenName = screenName
     }
+    fun setConsent(var1:  MutableMap<Any, Any>) {
```

Comment:
```suggestion

    fun setConsent(var1:  MutableMap<Any, Any>) {
```
Nitpick about spacing.